### PR TITLE
Clarify backplane facing capacity - consumed capacity

### PR DESCRIPTION
**Pull request by rohit-rp**
### Change Scope

Clarifying [/components/component/integrated-circuit/backplane-facing-capacity/state/consumed-capacity](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-platform.html#components-component-integrated-circuit-backplane-facing-capacity-state-consumed-capacity) to refer to the capacity consumed by traffic on the front panel ports

Clarifying [/components/component/integrated-circuit/backplane-facing-capacity/state/total](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-platform.html#components-component-integrated-circuit-backplane-facing-capacity-state-total)
and [/components/component/integrated-circuit/backplane-facing-capacity/state/total-operational-capacity](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-platform.html#components-component-integrated-circuit-backplane-facing-capacity-state-total-operational-capacity) to refer to the connection between `INTEGRATED_CIRCUIT` and `FABRIC` component identities.
The word `link` can be confused with an `interface` or `port`, when it actually means a `fabric link` or `SerDes link`

### Platform Implementations

 * Arista: [Document](https://www.arista.com/en/support/toi/eos-4-23-1f/14398-7500-series-fabric-capacity-monitor)
 
 * Cisco: [Document](https://www.cisco.com/c/en/us/td/docs/ios_xr_sw/iosxr_r3-8/general/man_pages/crpkg38/ar38fabr.pdf) Page 40

* [JunOS - show interface statistics fabric](https://www.juniper.net/documentation/us/en/software/junos/interfaces-ethernet-switches/topics/ref/command/show-interfaces-statistics-fabric-qfx-series.html#show-interfaces-statistics-fabric__d58076e851)

### CODE REVIEW

File: release/models/platform/openconfig-platform-integrated-circuit.yang [hunk 1]
```diff
@@ -102,16 +108,16 @@ module openconfig-platform-integrated-circuit {
...
       oc-ext:telemetry-on-change;
     }
 
     leaf consumed-capacity {
       type uint64;
       units "bits per second";
       description
-        "Backplane-facing capacity that is consumed by front-panel ports that are connected
-        to the integrated circuit and are operationally up.";
+        "Backplane-facing capacity that is consumed by the traffic on front-panel ports
```
Comment (robshakir) [inline release/models/platform/openconfig-platform-integrated-circuit.yang:119]: It's not strictly the traffic only those ports, right? If I have 400G front-panel port this consumes 400G, not whatever the traffic load on that port is at this exact moment in time.

Comment (dplore) [reply release/models/platform/openconfig-platform-integrated-circuit.yang:119]: Rohit and I discussed an issue brought up by implementors of this model.  The purpose of this PR is to explore and  clarify what the intent of this leaf is.   The modern, modular form factor packet based devices that I am familiar with only consume fabric capacity based on packets or cells sent over the fabric.  Bringing a front panel port up or down does not change the capacity that is in consumed between the INTEGRATED_CIRCUIT (IC) and the FABRIC.  

@rohit-rp I added a Juniper reference to the PR description.  Can you look for better references for Arista and Cisco?  I could not find information regarding 'consumed capacity' of IC to FABRIC in the references provided.

Comment (LimeHat) [reply release/models/platform/openconfig-platform-integrated-circuit.yang:119]: In Nokia SRLinux we provide this utilization data per direction (ingress/egress): https://documentation.nokia.com/srlinux/23-10/books/data-model-reference/srl_nokia-platform_0.html#d6179

**Comment by dplore** [comment]
After further review of the operational use cases for this leaf, it seems the desire is to get the 'subscription ratio' on a `LINECARD` between the capacity of enabled front panel ports and the capacity of the `LINECARD` connection to the `FABRIC`.    

So rather than using the word 'consumed', we should update this to use 'ratio of a LINECARD front panel interface bandwidth capacity to the bandwidth capacity of the connection between the LINECARD and FABRIC'.

**Pull request closed without merging**